tools/youtube_crawler.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+import os.path as osp
+import re
+
+try:
+    import requests
+    from requests.exceptions import SSLError
+except ImportError:
+    print('Install requests first.')
+
+
+def get_page(url):
+    while True:
+        try:
+            page = requests.get(url).text
+            return page
+        except SSLError:
+            print(f'Connection error {url}.')
+            continue
+
+
+def get_title(page):
```

Comment:
Actually, you can use youtube-dl to get metadata like title or descriptions or subtitles. No need to write your own code.

Reply:
Right, I just missed that `--get-title` args.